PassIO/templates/editevent.html

```diff
@@ -22,7 +22,8 @@
   {% include 'navbar.html' %}
```

Comment:
editevent.html looks great - easy to use with current field values displayed in each box and user just need to change the current value to update!